domains/certificates/Query.API/API/Query.API/ApiModels/Responses/Contract.cs

```diff
@@ -29,13 +30,21 @@ public class Contract
     /// </summary>
     public long Created { get; set; }
 
+    /// <summary>
+    /// Metering point type of the contract. Can be Production or Consumption
+    /// </summary>
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    [JsonPropertyName("meteringPointType")]
```

Comment:
why specify the json property name here?

Reply:
deleted